docs/sources/operators-guide/configure/configure-out-of-order-samples-ingestion.md

```diff
@@ -1,31 +1,21 @@
 ---
 aliases:
-  - /docs/mimir/latest/operators-guide/configuring/configuring-out-of-order-samples-ingestion/
+  - /docs/mimir/latest/operators-guide/configure/configure-out-of-order-samples-ingestion/
```

Comment:
I think we need to keep the old alias too, unless this will be backported to 2.2 branch (and in that case we don't need any alias at all because it's a new page).

Reply:
This will be backported to 2.2. Should I remove the alias then?

Reply:
Yes, because it's a new page that didn't exist in 2.1 doc (current "latest") at all.